src/unitxt/processors.py

```diff
@@ -75,6 +75,15 @@ def process(self, instance):
         return splitted[0].strip()
 
 
+class ConvertToBoolean(BaseFieldOperator):
+    def process(self, instance):
+        if any(w in str(instance).strip().lower() for w in ["yes", "right", "correct"]):
+            return "TRUE"
+        if any(w in str(instance).strip().lower() for w in ["no", "not", "wrong"]):
+            return "FALSE"
+        return instance.upper()
```

Comment:
oh IIUC its to catch the "true" and "false" lowercase outputs.. maybe a more explicit way would be to add "true" and "false" to the respective lists defined just above ..

Reply:
Also, a question: values that are not true or false, like "interesting" in your tests, do we want them perhaps converted to some fixed value like "unknown"?

Reply:
right, the task is essentially a binary classification task, where models answer questions based on passage; the references (the "answer" column) in the attached screenshot, are treated as strings in the (existing) accuracy metric in this case, that's why I'm simply aiming at precisely this set: {"TRUE", "FALSE"}

added "true" and "false" to the list, as you suggested, and changed any other return value to "OTHER", since it obviously won't match one of the {"TRUE", "FALSE"}, and is better readable

<img width="706" alt="image" src="https://github.com/IBM/unitxt/assets/38282865/d4a13571-ec9c-4e70-93a7-81dea47ae086">